aws/ec2/ec2/vm.go

```diff
@@ -2,13 +2,55 @@ package ec2
 
 import (
 	"github.com/DataDog/test-infra-definitions/aws"
+	"github.com/DataDog/test-infra-definitions/command"
+	"github.com/DataDog/test-infra-definitions/common/config"
 	"github.com/DataDog/test-infra-definitions/common/utils"
 
 	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/ec2"
 	"github.com/pulumi/pulumi-command/sdk/go/command/remote"
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
 )
 
+type VM struct {
+	context    *pulumi.Context
+	runner     *command.Runner
+	aptManager *command.AptManager
```

Comment:
Should be public and using the `PackageManager` interface

Reply:
Will address in a separate PR if it's ok for you